README.md

```diff
@@ -0,0 +1,68 @@
+# bdcat_notebooks
+Example Jupyter notebook source files for the BioData Catalyst consortium. These notebooks are published to the
+[Terra notebook environment](https://support.terra.bio/hc/en-us/articles/360027237871-Terra-s-Jupyter-Notebooks-environment-Part-I-Key-components).
+
+Jupyter notebook `.ipynb` files are generated from source files with the
+[callysto](https://github.com/xbrianh/callysto) library. 
+
+## Source File Schema
+The first line of each notebook source file contains the publish directive, a string with the following format:
+```
+# publish to "{my workspace name}" "{my notebook name}"
+```
+where `{my workspace name}` should be replaced with the name of the workspace, and `{my notebook name}` should be
```

Comment:
Does it support workspaces with spaces, periods, (), [], and {}? If there's any limitations, make note of that.

Reply:
This is a good idea, but I'm going to leave it blank for now:
  - This is developer tooling, and developers will need to deal with issues as they arise (such as forbidden characters in workspace names)"
  - I don't know which characters are forbidden. The readme should be updated by developers as information becomes available.